### DISCUSSION TITLE
打开再关闭后，会有进程驻留

### DISCUSSION THREAD

**Discussion by paroc108** [lyswhut/lx-music-desktop#932] [Q&A]
![image](https://user-images.githubusercontent.com/96353174/181767961-1d16d2c3-bfe2-4b0b-94e4-4904f4657c2d.png)
打开再关闭后，会有进程驻留

**Comment by lyswhut** [comment]
你这是程序运行异常或者开启了最小化托盘导致的吧？正常运行的情况下退出不会出现残留进程的情况
